src/org/toml/lang/annotator/TomlAnnotator.kt

```diff
@@ -0,0 +1,19 @@
+package org.toml.lang.annotator
+
+import com.intellij.lang.annotation.AnnotationHolder
+import com.intellij.lang.annotation.Annotator
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiWhiteSpace
+import com.intellij.psi.util.PsiTreeUtil
+import org.toml.lang.core.psi.TomlInlineTable
+
+public class TomlAnnotator : Annotator {
+    override fun annotate(element: PsiElement, holder: AnnotationHolder) {
+        if (element is TomlInlineTable) {
+            val whiteSpaces = PsiTreeUtil.findChildrenOfType(element, PsiWhiteSpace::class.java)
+            if (whiteSpaces.any { it.text.any { it == '\n' || it == '\r' } }) {
```

Comment:
Line breaks are normalized to `\n` by IntelliJ IDEA, so you could use `'\n' in it.text`